packages/@expo/cli/src/prebuild/resolveTemplate.ts

```diff
@@ -113,76 +114,70 @@ export async function resolveTemplateArgAsync(
   appName: string,
   template: string,
   templatePath?: string
-) {
-  let repoInfo: RepoInfo | undefined;
+): Promise<string> {
+  assert(template, 'template is required');
```

Comment:
confirmed that `template` always has a value and we can simplify the following code by removing the `if (template) { ... }` block